crates/hulks_os_version/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub static HULKS_OS_VERSION: &str = "5.3.0";
```

Comment:
There is a constant `SDK_VERSION` in the repository crate, which has the same value as the HULKs-OS version. I think you can use it instead of creating this specific new crate.

Reply:
Yes, please do so. Julian's PR got merged earlier and introduced a new crate for constants with a different name that you can use. Just put your constant in there.